example-llm-workflows/README.md

```diff
@@ -0,0 +1,483 @@
+# Running and Replaying AI Chains in Wasm On a Private, Locally Hosted LLM
+
+_Author_: [Zeeshan Lakhani][zeeshan-lakhani]
+
+## Outline
+
+- [Quickstart](#quickstart)
+- [Expanded Tutorial and Background](#expanded-tutorial-and-background)
+- [Outputs and Discussion](#outputs-and-discussion)
+- [Foundations for privacy](#foundations-for-privacy)
+- [What is Homestar in a Nutshell?](#what-is-homestar-in-a-nutshell)
+
+## Quickstart
+
+You can run this project completely in your console (shell). It consists of running a forked [Homestar][homestar] node (read more about Homestar in [What is Homestar in a Nutshell?](#what-is-homestar-in-a-nutshell)) and a local version of the [EveryCLI][every-cli] utility, a tool for working with the [Everywhere Computer][everywhere-computer], which is an interface for running workflows composed of [Wasm][wasm] / [Wasi][wasi] functions on Homestar.
+
+Let's first unzip [our tar file][tar-link] containing binaries and what's needed to run our demos, including workflow JSON files and [LLaMA 3][llama-3] models (created from [llama.cpp][llama.cpp]).
+
+```console
+tar -xf haii.tar.gz -C <target/directory>
+cd <target/directory>/example-llm-workflows
+```
+
+Everything going forward should take place within the `example-llm-workflows` folder.
+
+In one window, let's run the forked Homestar node (built with LLaMA 3 bindings) while also setting some environment variables:
```

Comment:
```suggestion
In one console window, run the forked Homestar node (built with LLaMA 3 bindings) while also setting some environment variables:
```
Keeping it consistent with the next "window" mention. Also, the "let's" isn't needed as you are directly giving instructions.